File: Peinture/src/main/java/com/example/peinture/Entities/Produit.java
package com.example.peinture.Entities;

import com.example.peinture.Entities.Categorie;
import lombok.*;

import javax.persistence.*;

@Entity
@NoArgsConstructor @AllArgsConstructor @Getter @Setter @ToString
public class Produit {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nom;
    private Double prix;
    private String couleur;
    private Double capacite;
    private String marque;
    private String imageUrl;



    public Produit(String nom, Double prix, String couleur, Double capacite, String marque,String imageUrl) {
        this.nom = nom;
        this.prix = prix;
        this.couleur = couleur;
        this.capacite = capacite;
        this.marque = marque;
        this.imageUrl = imageUrl;

    }
}

File: Frontend/HappyPaintApp/src/app/model/produit.ts
export class Produit{
    public id : string;
    public nom : string;
    public prix : number;
    public couleur : string;
    public capacite : number;
    public marque : string;
    public imageUrl : string;

    

    constructor(){
        this.id = '';
        this.prix = 0;
        this.nom = '';
        this.capacite = 0;
        this.couleur ='';
        this.marque = '';
        this.imageUrl = '';
    }
}
